extensions/applicationinsights-clickanalytics-js/scripts/publishReleaseToCdn.ps1

```diff
@@ -0,0 +1,594 @@
+            }
+        } else {
+            Get-AzureRmStorageAccount -ErrorAction SilentlyContinue | Out-Null
+        }
+
+        Log-Errors $false
+
+        #Get-AzureRmSubscription -SubscriptionId $subscriptionId -ErrorAction SilentlyContinue
+        foreach ($eacherror in $Error) {
+            if ($eacherror.Exception.ToString() -like "* Login-AzureRmAccount*") {
+                $loggedIn = $false
+
+                Log "Logging in... Atempt #$($attempt + 1)"
+                $Error.Clear()
+                Login-AzureRMAccount -ErrorAction SilentlyContinue 
+                Log-Errors $false
+                break
+            } elseif ($eacherror.Exception.ToString() -like "* Connect-AzureRmAccount*") {
+                $loggedIn = $false
+
+                Log "Connecting... Atempt #$($attempt + 1)"
+                $Error.Clear()
+                if ([string]::IsNullOrWhiteSpace($global:subscriptionId) -ne $true -and (IsGuid($global:subscriptionId) -eq $true)) {
+                    Connect-AzureRmAccount -ErrorAction SilentlyContinue -Subscription $global:subscriptionId | Out-Null
+                } else {
+                    Connect-AzureRmAccount -ErrorAction SilentlyContinue | Out-Null
+                }
+
+                Log-Errors $false
+                break
+            } else {
+                $loggedIn = $false
+                Log-Warning "Unexpected failure $($eacherror.Exception)"
+            }
+        }
+
+        $attempt ++
+    }
+
+    $Error.Clear()
+}
+
+Function AddReleaseFile(
+    $files,
+    [string] $releaseDir,
+    [string] $name
+) {
+    $sourcePath = (Join-Path $releaseDir -ChildPath ($name))
+
+    if (-Not (Test-Path $sourcePath)) {
+        Log-Warning "Missing expected source file '$sourcePath'";
+        exit         
+    }
+
+    Log " - $sourcePath"
+    $files.Add($name, $sourcePath)
+}
+
+Function GetReleaseFiles
+{
+    if ([string]::IsNullOrWhiteSpace($jsSdkDir) -eq $true) {
+        Log-Warning "Invalid JS Sdk Path"
+        exit
+    }
+
+    Log "Releasing from : $jsSdkDir"
+
+    # find version number
+    $packageJsonPath =  Join-Path $jsSdkDir -ChildPath "package.json"
+    if (-Not (Test-Path $packageJsonPath)) {
+        Log-Warning "'$packageJsonPath' file not found, please enter the top JSSDK directory.";
+        exit
+    }
+
+    $packagesJson = (Get-Content $packageJsonPath -Raw) | ConvertFrom-Json
+    $version = $packagesJson.version;
+
+    Log "Version        : $version"
+
+    # check if the minified dir exists
+    $jsSdkSrcDir = Join-Path $jssdkDir -ChildPath "browser\";
+
+    if (-Not (Test-Path $jsSdkSrcDir)) {
+        Log-Warning "'$jsSdkSrcDir' directory doesn't exist. Compile JSSDK first.";
+        exit
+    }
+
+    $files = New-Object 'system.collections.generic.dictionary[string,string]'
+
+    Log "Adding files";
+    AddReleaseFile $files $jsSdkSrcDir "ai.clck.$version.js"
+    AddReleaseFile $files $jsSdkSrcDir "ai.clck.$version.js.map"
```

Comment:
We should add the .gbl. and .cjs files (same as core publish files) -- these should be added to the debug plugin as well.

The gbl files are specifically for sharepoint support